rubygems-update.gemspec

```diff
@@ -14,7 +14,12 @@ Gem::Specification.new do |s|
   repository of Gems that allows you to publish a Gem that can be shared and used by other
   developers. See our guide on publishing a Gem at guides.rubygems.org"
   s.homepage = "https://guides.rubygems.org"
-  s.metadata = { "source_code_uri" => "https://github.com/rubygems/rubygems" }
+  s.metadata = {
+    "source_code_uri" => "https://github.com/rubygems/rubygems",
+    "bug_tracker_uri" => "https://github.com/rubygems/rubygems/issues",
+    "changelog_uri" => "https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md",
+    "funding_uri" => "https://rubycentral.org/#/portal/signup",
```

Comment:
@indirect I'm not sure this is correct URI for funding Ruby Central. Can you look this?

Reply:
Yes, that is the correct URI for the membership signup page. I don't know why, but [Ghost](https://ghost.org) uses javascript to show the signup page over the regular home page :(

Reply:
Got it. Thanks.